libcloudforensics/providers/gcp/internal/common.py

```diff
@@ -35,6 +35,7 @@
 LOGGER = logging.getLogger()
 RETRY_MAX = 10
 REGEX_DISK_NAME = re.compile('^(?=.{1,63}$)[a-z]([-a-z0-9]*[a-z0-9])?$')
+COMPUTE_NAME_LIMIT = 63
```

Comment:
shouldn't this go in `compute.py`?

Reply:
this is for the instance name generation method